scripts/initiate_rep_set.sh

```diff
@@ -3,7 +3,7 @@
 
 i=0
 while [ $i -lt 20 ]; do
-  curl -s "${ES_URL}/_cluster/health?wait_for_status=yellow&timeout=10s" > /tmp/output
+  curl -s "http://${ES_USERNAME}:${ES_PASSWORD}@${ES_HOST}:${ES_PORT}/_cluster/health?wait_for_status=yellow&timeout=10s" > /tmp/output
```

Comment:
Is it ok to handle the case where we don't have a password in the python code but not here?

Reply:
no it's not, done